test/src/macosTest/kotlin/io/realm/shared/EncryptionTests.kt

```diff
@@ -0,0 +1 @@
+../../../../../androidTest/kotlin/io/realm/shared/EncryptionTests.kt
```

Comment:
I thought the whole `shared` directory was symlinked. Would that be possible?

Reply:
Yes it is, I am going to address it on another PR